tee-worker/omni-executor/executor-worker/src/main.rs

```diff
@@ -90,8 +89,47 @@ async fn main() -> Result<(), ()> {
 
 	match cli.cmd {
 		Commands::Run(args) => {
-			let args_string = std::env::args().collect::<Vec<String>>().join(" ");
-			info!("Executing: {}", args_string);
+			let env_vars = {
+				let vars = [
+					("OE_PARENTCHAIN_URL", "parentchain_url"),
+					("OE_ETHERUM_URL", "ethereum_url"),
+					("OE_SOLANA_URL", "solana_url"),
+					("OE_BSC_URL", "bsc_url"),
+					("OE_BSC_TESTNET_URL", "bsc_testnet_url"),
+					("OE_ALCHEMY_KEY", "alchemy_key"),
+					("OE_PUMPX_SIGNER_URL", "pumpx_signer_url"),
+					("OE_PUMPX_API_BASE_URL", "pumpx_api_base_url"),
+					("OE_BINANCE_API_KEY", "binance_api_key"),
+					("OE_BINANCE_API_SECRET", "binance_api_secret"),
+					("OE_BINANCE_API_BASE_URL", "binance_api_base_url"),
+					("OE_WORKER_BASE_URL", "worker_base_url"),
+				];
+
+				let mut env_map = std::collections::HashMap::new();
+
+				for (env_name, var_name) in vars {
+					let value = Some(std::env::var(env_name).unwrap_or_default());
+					env_map.insert(var_name, value);
+				}
+
+				let args_string = std::env::args().collect::<Vec<String>>().join(" ");
+				info!("Executing: {}", args_string);
+
+				for (name, value) in &env_map {
+					if *name != "alchemy_key"
+						&& *name != "binance_api_key"
+						&& *name != "binance_api_secret"
+					{
+						if let Some(val) = value {
+							if !val.is_empty() {
+								info!("Environment variable {}: {}", name, val);
```

Comment:
Can we also log empty ?

Reply:
Maybe one more step: not only log empty, but also move the default value assignment here (if any). e.g.: there is default value for binance_api_url, move the default value assignment here together.